packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -218,21 +214,6 @@ export class PendingStateManager implements IDisposable {
         assert(pendingState.type === "message", 0x169 /* "No pending message found for this remote message" */);
         this.pendingStates.shift();
 
-        // Processing part - Verify that there has been no data corruption.
```

Comment:
consider splitting these changes into separate PR and get it in sooner (to keep your main PR smaller).

Reply:
https://github.com/microsoft/FluidFramework/pull/13917